### ISSUE TITLE
Refresh Token when user log in with Google

### ISSUE THREAD

**Issue by marvi14** [Armax/Pokemon-GO-node-api#255]
Did someone manage to renew token from an expired accessToken? the thing is that renewing the token with username and password is possible, but when the users logs in with google, the google AuthCode is expired, and i don't want the user to relog with google again. Any ideas?

**Comment by ameyer** [comment]
I never got it working. I had it setup where it would login every 30min - Just using a timeout

**Comment by marvi14** [comment]
Yeah, for now i will just user username, password and provider until a solution is found
